src/Tribe/Commerce/Currency.php

```diff
@@ -367,6 +383,38 @@ protected function get_provider_symbol( $provider, $object_id ) {
 		return $this->get_currency_symbol( $object_id );
 	}
 
+	/**
+	 * Returns he cost applying separators from Woo or EDD depends on the provider.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $provider
+	 * @param string $cost
+	 *
+	 * @return string
+	 */
+	protected function get_provider_cost( $provider = '', $cost = '' ) {
+		if ( ! class_exists( $provider ) ) {
+			return $cost;
```

Comment:
curious about just returning the cost -- we have some `Cost_Utils` in tribe common, with some cost-formatting methods of our own. should we try to format the cost using those methods if no provider methods are available? 

sorry if this was addressed already or not relevant

Reply:
Thanks for asking at this stage the cost has been already formatted by `number_format` so that's why is a `string` and not a `number` so we are just falling back to the initial formatting of cost. Not sure if is worth it.